Source/Lib/Codec/EbInitialRateControlProcess.c

```diff
@@ -342,15 +342,17 @@ void DetectGlobalMotion(
 	pictureControlSetPtr->isPan = EB_FALSE;
 	pictureControlSetPtr->isTilt = EB_FALSE;
 
-
-	// If more than PAN_LCU_PERCENTAGE % of LCUs are PAN
-	if ((totalPanLcus * 100 / totalCheckedLcus) > PAN_LCU_PERCENTAGE) {
-		pictureControlSetPtr->isPan = EB_TRUE;
-	}
-
-	if ((totalTiltLcus * 100 / totalCheckedLcus) > PAN_LCU_PERCENTAGE) {
-		pictureControlSetPtr->isTilt = EB_TRUE;
-	}
+    if (totalCheckedLcus) {
```

Comment:
combine two if (totalCheckedLcus)
better with if (totalCheckedLcus != 0)? And all the rest if check.